tck/README.adoc

```diff
@@ -43,13 +43,23 @@ Scenario: Returning a single node
       | n  |
       | () |
 ----
-<1> This step specifies the required initial state of the graph. `<GRAPH_NAME>` is the name of one of the specified graphs for initial states (read more below).
+[source,gherkin]
+.This scenario illustrates the effects of executing a query `Q` - matching all nodes and returning one of them - on the graph <GRAPH_NAME>.
+----
+Scenario: Indexing a list with a string
+    Given any graph // <1>
+    When executing query: WITH [0] AS expr, 'x' AS idx RETURN expr[idx] // <4>
+    Then a TypeError should be raised at runtime: ListElementAccessByNonInteger // <7>
+----
```

Comment:
I think that these numbers at the end of each line are very important..but hard to read. Could they be made bold or something so that they stand out a bit more?